torch/export/_remove_auto_functionalized_pass.py

```diff
@@ -73,15 +26,20 @@ def unsafe_remove_auto_functionalized_pass(
     and modifies the calling EP inplace to have the original mutator op.
     This pass doesn't perform safety checks to make sure that this inplace mutation is safe.
     """
-    auto_functionalize_nodes: List[torch.fx.Node] = []
-    for module in ep.graph_module.modules():
-        if not isinstance(module, torch.fx.GraphModule):
-            continue
-        for node in ep.graph.nodes:
-            if node.op == "call_function" and node.target is auto_functionalized:
-                auto_functionalize_nodes.append(node)
 
     with ep.graph_module._set_replace_hook(ep.graph_signature.get_replace_hook()):
-        _remove_auto_functionalization_from_graph_helper(ep, auto_functionalize_nodes)
+        for module in ep.graph_module.modules():
```

Comment:
Also, this PR is getting kind of big. There are things that we could have split off:
- this export piece is splittable by itself
- the test movement into test_functionalize.py is splittable

It's up to you if you want to split it at this point

Reply:
This PR by itself is relatively risky so splitting it up can help protect against rebase pain if reverts happen

Reply:
ok yes splitting is good.

Reply:
- i can do that this part can go before
- the test movement thise we can land quicker then we land the big change.

Reply:
done split and added two more PRs this one on top of them now.